src/content-tags-drawer/data/api.js

```diff
@@ -70,19 +70,14 @@ export async function getContentTaxonomyTagsCount(contentId) {
 }
 
 /**
- * Fetch meta data (eg: display_name) about the content object (unit/compoenent)
+ * Fetch meta data (eg: display_name) about the content object (unit/component)
```

Comment:
nit: Unit keys start with `lct:`, so aren't explicitly handled here.. maybe all these `(unit/component)` comments should be `(course/library component)` instead?

Reply:
This is from the legacy studio/course outline, and "unit" stands for Course Units (we can tag them). More on that on the other comments.